harvest/api/yahoo.py

```diff
@@ -92,8 +92,7 @@ def fetch_price_history( self,
         df = yf.download(symbol, period=period, interval=get_fmt, prepost=True)
         debug(df)
         df = self._format_df(df, symbol)
-        df = df.loc[start:end]
-        
+        df = df.loc[start.replace(tzinfo=pytz.UTC):end.replace(tzinfo=pytz.UTC)]
```

Comment:
Some tests where failing because `start` and `end` where not UTC timezone aware and the dataframe is UTC timezone aware.